eng/tools/versioning/increment.js

```diff
@@ -42,6 +42,11 @@ async function main(argv) {
     (packageSpec) => packageSpec.packageName.replace("@", "").replace("/", "-") == artifactName
   );
 
+  if (targetPackage.versionPolicyName == "management") {
```

Comment:
Lets understand the versioning strategy before we disable this automation. I'd like to make sure we are consistent.

Reply:
@weshaggard @praveenkuttappan We prefer not to update mgmt plane SDK is because we plan to have some version strategy in the future after the SDK generator is GA. for example, we will probably have to consider the swagger tag. if that tag is preview tag, then the SDK should be preview version, if that tag is stable tag, but it contains swagger file in preview version, we should also put it as preview version. Another reason is that mgmt. plane doesn't like data plane. most of the code in mgmt. plane are autogenerated. Once the SDK is released. we will not have un-released changes for a dev preview until next release. So most of the alpha version SDK would be exactly the same as the published ones. Which we believe is a little redundant.

Reply:
@qiaozha as I understand the mgmt plane libraries are fully auto-generated but will there ever be any changes made to the library outside of the automation with the auto-generated code? Does the version number always get incremented every time we submit a PR with the new auto-generated code? The reason I ask this is because I want to make sure there are no changes in the repo to the library that doesn't also contain a version number change. We need to ensure what we build in the repo does not conflict with what is on npm and as long as the code is exactly the same as what is published on npm then I agree it might not be necessary to increment the version until there is a change.

I will also state that even for data-plane we only publish the alpha versions to npm if there are changes we don't publish packages which contain no changes.

Reply:
@weshaggard Yes, there will be small changes made by our tool that is outside of code generator scope such as version bump, change log and probably some tests in the future, but these changes will be released along with generated code.  
> I will also state that even for data-plane we only publish the alpha versions to npm if there are changes we don't publish packages which contain no changes.

I am glad to see that's guranteed. Then I guess it couldn't be hurt to add alpha version release for mgmt plane if for consistence purpose.

Reply:
@qiaozha do you have any docs or pointers to how and when version numbers get updated for the mgmt libraries? It sounds like that part is still manual so how do we prevent issues where some changes come into the library before we increment the version number? That is part of the reason why we have automation that always increments the version of the data-plane libraries as soon as we do a release so we don't ever have a case where we have conflicting libraries we build in our repo vs what is on npm.

Reply:
In mgmt plane, when we review PR, we will not merge the PR until we see version bump change come along with other changes. For track1 SDK, we will bump the version manually, For track2, You may refer to this link https://github.com/dw511214992/track2-js-sdk-document/blob/main/Details-of-Automation-Tool.md#generate-changelog-and-bump-version-for-azure-track2-js-sdk-1 for more information. I think both way will not cause conflicts between what we build in our repo and what is on npm as long as we keep in mind to bump a version whenever merge a PR.  Hope this helps.

Reply:
@weshaggard I just been informed that in the track2 JS SDK, the first week of each month is reserved for JS core libraries update and release. They will file a PR to update all the core libraries dependencies inside each RP's SDK to make `rush update` work. All the releases for RP SDK will be stop before they actually released the new core libraries. see pr #16738 for an example. In such case, the premise of  "we will not merge the PR until we see version bump change come along with other changes." is no long valid. And we would like to keep the alpha releases for mgmt plane in case there's something breaking. @praveenkuttappan You may close this PR. Thanks for all your help.

Reply:
@qiaozha I am confused now. How will alpha versioning work properly if we are not going to increase version before any change is made in package after previous version is released? For e.g. Let's assume you  have released @azure/arm-compute 10.0.0 to npm and new changes were made in package to update dependency or any other change. Alpha version will still be created as 10.0.0-alpha<build-date> and by semver comparison this is lower than 10.0.0 when it is supposed to be alpha of 11.0.0.

This is exactly why we are trying to understand versioning workflow for mgmt packages and how this integrated properly into current daily alpha builds.

Reply:
I cannot access https://github.com/dw511214992/track2-js-sdk-document/blob/main/Details-of-Automation-Tool.md#generate-changelog-and-bump-version-for-azure-track2-js-sdk-1. I think we have reached a point where we should setup a meeting to try and come up with the correct versioning strategy for these libraries.